packages/core/package.json

```
@@ -17,6 +17,10 @@
       "require": "./dist/lib/index.cjs",
       "import": "./dist/lib/index.js"
     },
+    "./types": {
+      "require": "./dist/types.d.cts",
+      "import": "./dist/types.d.ts"
```

my thinking is we should point to the JS files instead since I would imagine that using `.*ts` files here will muck up Node's module resolution. i think `.*js` is the move since `tsup` is properly generating those files (even though they're empty) and [modern TS should pick up on the co-located declaration files](https://www.typescriptlang.org/docs/handbook/modules/reference.html#example-subpaths-conditions-and-extension-substitution).

```suggestion
      "require": "./dist/types.d.cjs",
      "import": "./dist/types.d.js"
```